frontend/src/features/properties/map/acquisition/add/AcquisitionProperties.tsx

```diff
@@ -23,6 +29,9 @@ export const AcquisitionProperties: React.FunctionComponent = () => {
                 onSelectedProperty={(newProperty: IMapProperty) => {
                   const formProperty = AcquisitionPropertyForm.fromMapProperty(newProperty);
                   push(formProperty);
+                  if (values.properties?.length === 0) {
+                    formikProps.setFieldValue(`region`, formProperty.region);
+                  }
```

Comment:
Can we double check this logic as well? The **push()** call in line 31 should modify `values.properties` by inserting the new property so I find it odd that we could ever check for `values.properties.length === 0` the line after...

Reply:
unless the update occurs asynchronously and then we risk a race condition

Reply:
We want to use region of only first property to be set in ministry region so the check. The push() will happen every time but can be moved to after the check and setting the region.